### PR TITLE
Track2 : Add pojo and entity for app service library

### PR DESCRIPTION
**Pull request by Flanker32**
Add pojo and entity for app service library

### CODE REVIEW

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/entity/WebAppDeploymentSlotEntity.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+@Getter
+@SuperBuilder(toBuilder = true)
+public class WebAppDeploymentSlotEntity {
+    private String id;
+    private String name;
+    private String webappName;
+    private String resourceGroup;
+    private String subscriptionId;
+    private String appServicePlanId;
```
Comment (andxu) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/entity/WebAppDeploymentSlotEntity.java:25]: extract common :
    private String id;
    private String name;
    private String webappName;
    private String resourceGroup;
    private String subscriptionId;

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/model/DeployType.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    public static final DeployType JAR = new DeployType("jar");
+    public static final DeployType EAR = new DeployType("ear");
+    public static final DeployType JAR_LIB = new DeployType("lib");
+    public static final DeployType STATIC = new DeployType("static");
+    public static final DeployType SCRIPT_STARTUP = new DeployType("startup");
+    public static final DeployType ZIP = new DeployType("zip");
+
+    private String value;
+
+    public List<DeployType> values() {
```
Comment (andxu) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/model/DeployType.java:28]: enum?

File: azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/tools/common/model/ResourceGroup.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ */
+package com.microsoft.azure.tools.common.model;
+
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@SuperBuilder(toBuilder = true)
+public class ResourceGroup {
+    private String name;
```
Comment (andxu) [inline azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/tools/common/model/ResourceGroup.java:14]: id?

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/entity/WebAppDeploymentSlotEntity.java (hunk 1 above)
Comment (Flanker32) [reply azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/entity/WebAppDeploymentSlotEntity.java:25]: May sync and do this later in common lib

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/model/DeployType.java (hunk 2 above)
Comment (Flanker32) [reply azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/model/DeployType.java:28]: We may discuss whether to use enum or Constant later and refactor in single pr

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/model/WebContainer.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+import java.util.Arrays;
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+public class WebContainer {
+    private String value;
+
+    public static final WebContainer JAVA_8 = new WebContainer("java 8");
```
Comment (andxu) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/model/WebContainer.java:20]: JAVA_SE

**Review by andxu — APPROVED** [review]

**Pull request merged**